metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -62,9 +60,12 @@ class DatahubClientConfig(ConfigModel):
     retry_max_times: Optional[int]
     extra_headers: Optional[Dict[str, str]]
     ca_certificate_path: Optional[str]
-    max_threads: int = 15
     disable_ssl_verification: bool = False
 
+    _max_threads_moved_to_sink = pydantic_removed_field(
```

Comment:
Is this a breaking change? 
Should we add this to the release note and also notify users?

Reply:
this helper maintains backwards compat by ignoring that field, so it's not a breaking change